src/GraphQL/Utilities/AstPrinter.cs

```diff
@@ -1,558 +0,0 @@
-using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Globalization;
-using System.Linq;
-using System.Linq.Expressions;
-using System.Numerics;
-using GraphQL.Language.AST;
-using GraphQL.Utilities.Federation;
-
-namespace GraphQL.Utilities
-{
-    /// <summary>
-    /// Prints a string representation of the specified AST document or node.
-    /// </summary>
-    public static class AstPrinter
```

Comment:
I am very glad that deleted this file. I never understood how `AstPrinter` works. Closures of closures in closures 😖 , expression resolvers 😮 , deeply nested fluent configs 😕 and A LOT os string allocations-concatenations.

💥 💥 💥